app/views/content/is-teaching-right-for-me/all-subjects.md

```diff
@@ -0,0 +1,298 @@
+---
+title: Look at the subjects you could teach
+description: |-
+  Look at the subjects you could teach and choose what teacher training course you would like to do.
+
+keywords:
+  - teacher training
+  - teaching maths
+  - teaching history
+  - teaching science
+  - teaching art
+  - art and design
+
+
+---
+
+Explore the A-Z of all the subjects you could choose for your teacher training course.
```

Comment:
Do we want to take out all and just say the subjects as I think we said we're not covering every subject in this page as we don't have info for all of them.

Reply:
Hm good question @MaxineCl the only one missing is ancient hebrew which i could add back in with very generic description. Or just change to 'A-Z of subjects...'

Reply:
@caitlin-reidy I think let's go with A-Z for now. That could also cover us if new subjects get added that we haven't covered.